.github/mergeable.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+mergeable:
+  - when: pull_request.*
+    name: "Changelog check"
+    validate:
+        - do: and
+          validate:
+            - do: dependent
+              changed:
+                file: 'src/**' 
```

Comment:
just curious: what do the double asterisk `**` mean here?

Reply:
no idea, but I'll guess that it means "all files _as well as files in subdirectories_"

Reply:
yes, this is one of the so called "glob" patterns I believe: https://en.wikipedia.org/wiki/Glob_(programming)